docuilib/yarn.lock

```diff
@@ -6069,7 +6069,7 @@ lodash.uniq@^4.5.0:
   resolved "https://registry.npmjs.org/lodash.uniq/-/lodash.uniq-4.5.0.tgz#d0225373aeb652adc1bc82e4945339a842754773"
   integrity sha512-xfBaXQd9ryd9dlSDvnvI0lvxfLJlYAZzXomUYzLKtUeOQvOP5piqAWuGtrhWeqaXK9hhoM/iyJc5AV+XfsX3HQ==
 
-lodash@^4.17.20, lodash@^4.17.21:
+lodash@4.x.x, lodash@^4.17.20, lodash@^4.17.21:
```

Comment:
`yarn.lock` should not change

Reply:
You are right I'm not sure about that checking it.

Reply:
I didn't managed to revert it, tried to checkout the commit and revert the file didn't managed to.